### ISSUE TITLE
Feature Request: Ability to adjust payload to suit (move payload.on to payload, change on/off command)

### ISSUE THREAD

**Issue by y8s** [mikejac/node-red-contrib-google-smarthome#206]
Many of the nodes I'm working with (Tasmota, TP-Link, Zigbee2MQTT) are expecting msg.payload to contain the command (on/off, true/false, 1/0) rather than looking for it in msg.payload.on. 

Similarly, those nodes also output their state as a msg.payload and not msg.payload.on 

I understand that `msg.payload.on bool` is the standard google trait state but with as many assistant nodes as I have, I'll have to add two more nodes to each to achieve what I need in this case. 

In short: it would be awesome to be able to customize the command output to make it msg.payload and have it listen for the same state inputs.

**Comment by ckhmer1** [comment]
In my opinion is quite difficult to add a general mechanism to change the output format. It easier to add a node after three Google one to arrange the different formats.

**Comment by y8s** [comment]
Difficult for whom? My formatting function nodes are all over the place. NORA nodes, for example, don't require nearly as much effort to use (just money). They have some secret sauce that just works properly with my other nodes.
